.github/instructions/mastg-test.instructions.md

```diff
@@ -0,0 +1,208 @@
+## Tests
+
+Tests are platform-specific and must be located under tests-beta/android/ or tests-beta/ios/, within the corresponding MASVS category. Their file names are the test IDs.
+
+Notes:
+
+- Tests with `platform: network` are still organized under the OS folder that the MASVS category belongs to (for example, Android network tests live under `tests-beta/android/MASVS-NETWORK/`).
```

Comment:
Readability-wise, it might be smoother to read after the `ls -l` "structure" example. Wider context->edge case